assets/scripts/util/street_analytics.js

```diff
@@ -61,46 +63,111 @@ const CAPACITIES = {
   'magic-carpet': { average: 2, potential: 3 }
 }
 
-const hasCapacityType = (type) => {
+const CAPACITIES_FOO = {
+  sidewalk: { average: 1500, potential: 1500 },
+  'drive-lane': { average: 1500, potential: 1500 },
+  'bike-lane': { average: 1500, potential: 1500 },
+  scooter: { average: 1500, potential: 1500 },
+  'light-rail': { average: 1500, potential: 1500 },
+  streetcar: { average: 18000, potential: 1500 },
+  'bus-lane': { average: 5000, potential: 8000 },
+  'magic-carpet': { average: 1500, potential: 1500 }
+}
+
+const CAPACITIES_BAR = {
+  sidewalk: { average: 5000, potential: 5000 },
+  'drive-lane': { average: 5000, potential: 5000 },
+  'bike-lane': { average: 5000, potential: 5000 },
+  scooter: { average: 5000, potential: 5000 },
+  'light-rail': { average: 18000, potential: 18000 },
+  streetcar: { average: 18000, potential: 20000 },
+  'bus-lane': { average: 5000, potential: 8000 },
+  'magic-carpet': { average: 2, potential: 3 }
+}
+
+const hasCapacityType = (type, source = '') => {
+  if (source === 'foo') {
+    return type in CAPACITIES_FOO
+  }
+  if (source === 'bar') {
+    return type in CAPACITIES_BAR
+  }
```

Comment:
This entire setup means we are needing to store data hard-coded. I think we need a different approach. Throughout Streetmix we tend to store such data as JSON. Let's come up with a way to store and reference this information from a JSON array of data, as opposed to hard-coding.

Reply:
I've consolidated the data relating to analytics capacities into the JSON object `SOURCE_LIST`, and stored that in `app/constants`. Let me know if you think we should separate this out further, like we do with translations, but I found that composing the object in the constants file made things very readable.